validation/shallow_water_model/shallow_water_jet_topography.jl

```diff
@@ -0,0 +1,167 @@
+using Oceananigans
+using Oceananigans.Models: ShallowWaterModel
+   
+Nx, Ny = 50, 128
+Lx, Ly = 2π,   10
+H₀     = 10
+α, β   = 0.1,  0.1
+Lb, Hb = α * Ly, β * H₀
+
+f, g, U  = 1, 1, -1.0
+Δη = f * U / g 
+
+grid = RectilinearGrid(size = ( Nx, Ny),
+                          x = (  0, Lx),
+                          y = (-Ly, Ly),
+                   topology = (Periodic, Bounded, Flat))
+
+η(x, y) = Δη * tanh(y)
+b(x, y) = Hb * tanh(y/Lb)
+
+h̄(x, y) = H₀ + η(x, y) + b(x, y)
```

Comment:
so this is the total fluid depth, right?

Reply:
Yes.  In order to get a reasonable result I needed to set `bathymetry = - b`.  This was not my first guess and I want to make sure I understand why this is the correct choice.  The PDF that I posted with the equations does not follow this convention, so I need to fix that up, after I understand this better.

Reply:
Yes, very good idea as clearly this is a point of confusion.

Reply:
I guess the docs are clear:

> The elevation of the bottom bathymetry, measured with respect to the free-surface at rest.

Reply:
Agreed!  Now to understand why this doesn't work like we expect.  Or, at least how i expect.

Reply:
Well there is a good chance the docs say one thing but the source code does something else!